tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -862,24 +863,24 @@ def test_uss_block_insert_with_indentation_level_specified(ansible_zos_module):
     finally:
         remove_uss_environment(ansible_zos_module)
 
-
-@pytest.mark.uss
-def test_uss_block_insert_with_doublequotes(ansible_zos_module):
-    hosts = ansible_zos_module
-    params = dict(insertafter="sleep 30;", block='cat \"//OMVSADMI.CAT\"\ncat \"//OMVSADM.COPYMEM.TESTS\" > test.txt', marker="// {mark} ANSIBLE MANAGED BLOCK", state="present")
-    full_path = TEST_FOLDER_BLOCKINFILE + inspect.stack()[0][3]
-    content = TEST_CONTENT_DOUBLEQUOTES
-    try:
-        set_uss_environment(ansible_zos_module, content, full_path)
-        params["path"] = full_path
-        results = hosts.all.zos_blockinfile(**params)
-        for result in results.contacted.values():
-            assert result.get("changed") == 1
-        results = hosts.all.shell(cmd="cat {0}".format(params["path"]))
-        for result in results.contacted.values():
-            assert result.get("stdout") == EXPECTED_DOUBLE_QUOTES
-    finally:
-        remove_uss_environment(ansible_zos_module)
+# Test case base on bug of dataset.blockifile 
```

Comment:
Do you have the validation task number (validation as the GH issue we will use to retest in the future) ? That is bc even if this states we need to fix it is not easy to identify which issue are we talking about.

Reply:
You can add the ZOAU issue number or GH validation task number